birdseye/__init__.py

```diff
@@ -191,6 +191,12 @@ def extract_values(iteration, path):
         session.commit()
 
     def __call__(self, func):
+        if inspect.isclass(func):
+            for name, meth in func.__dict__.items():
+                if inspect.ismethod(meth) or inspect.isfunction(meth):
+                    setattr(func, name, self.__call__(meth))
```

Comment:
Since `func` isn't really a function this is a bit confusing. Inside the first `if` set `cls = func` and use `cls` instead.